internal/policy/source/git_config_test.go

```diff
@@ -30,7 +30,6 @@ func TestSourceIsFile(t *testing.T) {
 		want bool
 	}{
 		{src: "", want: false},
-		{src: "foo", want: true},
```

Comment:
If it is intentional, would it be better to make it `want: false` instead of removing it?

Reply:
Yes, I think you're correct. I'll add this back and set it false.